cognee/infrastructure/databases/graph/networkx/adapter.py

```
@@ -254,6 +250,29 @@ async def load_graph_from_file(self, file_path: str = None):
             if os.path.exists(file_path):
                 async with aiofiles.open(file_path, "r") as file:
                     graph_data = json.loads(await file.read())
+                    for node in graph_data["nodes"]:
+                        try:
+                          node["id"] = UUID(node["id"])
+                        except:
```

Ruff

Do not use bare `except`

(E722)